tokio-uds/src/lib.rs

```diff
@@ -11,13 +11,16 @@ extern crate bytes;
 extern crate futures;
 extern crate iovec;
 extern crate libc;
+#[macro_use]
 extern crate log;
```

Comment:
There was no other use of logging macros in this library, I'm not sure if that was intentional or not. If it was I'll need to remove some `trace!` statements.

Reply:
This is fine.